osmaxx/excerptexport/static/excerptexport/scripts/maps/draw_controls.js

```diff
@@ -82,6 +89,41 @@ var draw_controls = function (map) {
             nameField.parentNode.appendChild(e);
         }
 
+        function getAllowedMaxSize(){
+            var selectedDetailLevel = jQuery('#id_detail_level');
+            var selectedValue = selectedDetailLevel.find(":selected").attr('value');
+            if (parseInt(selectedValue) === 60) {
+                return allowedMaxSizeSimplified;
+            }
+            return allowedMaxSize;
+        }
+
+        function isSimplifiedChosen() {
+            var selectedDetailLevel = jQuery(detailLevelID).find(":selected").attr('value');
+            return selectedDetailLevel === simplifiedSelectValue;
+        }
+
+        function animateUserToSwitchToLowerDetails() {
```

Comment:
As 'animate' is used in a different meaning within this function, consider a different verb for the function name. Maybe 'encourage'?